game/enums/eatenPowers.py

```diff
@@ -0,0 +1,20 @@
+"""This module contains the eaten powers class."""
+from enum import Enum, auto
+
+
+class EatenPowers(Enum):
+    """Direction class which contains the enum of all possible directions.
+
+    Attributes
+    ----------
+    empty_stomach : Enum
+        The value representing an empty stomach
+    basic_skeleton : Enum
+        The value representing a basic skeleton was eaten
```

Comment:
Grammar/complete sentences.
```suggestion
        The value representing a basic skeleton that was eaten
    damsel : Enum
        The value representing a damsel that was eaten
```